.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           command: cd ~ && ./local-setup/deployments/local-setup-instance/provisioning/run-bitcoin.sh
       - run:
```

Comment:
If we're in this land could we try to improve the steps declaration and replace all
`cd ~ && ./local-setup/deployments/local-setup-environment/provisioning` with `working_directory` property, e.g.:
```yaml
      - run:
          name: Install common tools
          working_directory: ~/local-setup/deployments/local-setup-environment/provisioning
          command: ./install-commons.sh
```

Reply:
Done.